ingestor-default/src/main/scala/za/co/absa/hyperdrive/ingestor/implementation/writer/parquet/ParquetPartitioningStreamWriter.scala

```diff
@@ -93,4 +94,19 @@ object ParquetPartitioningStreamWriter extends StreamWriterFactory {
       case _ => reportDateFormatter.format(LocalDate.now())
     }
   }
+
```

Comment:
You could define description/name/properties in a separate files. 
https://docs.scala-lang.org/tour/self-types.html